packages/admin-ui/server/getWebpackConfig.js

```diff
@@ -27,7 +25,6 @@ module.exports = function({ adminMeta, entry, outputPath }) {
         // since it doesn't have to compile these things with babel
         // note that the preconstruct aliases are also disabled on heroku to enable this
         // when we have static builds for the admin ui, we can remove this
-        ...(isHerokuEnv ? [/@keystone-alpha\/field/, /@arch-ui/] : []),
```

Comment:
This isn't necessary now because we have static building and we don't care as much about memory usage during building since it won't happen when the server is running

Reply:
Can also remove the comment above it.